openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -168,36 +168,38 @@ def has_override_value(name):
 
 def get_value_for_org(org, val_name, default=None):
     """
-    This returns a configuration value for a site configuration which has an org_filter that matches
-    what is passed in.
+    This returns a configuration value for a site configuration or microsite configuration
+    which has an org_filter that matches with the argument.
 
     Args:
-        org (str): Course ord filter, this value will be used to filter out the correct site configuration.
+        org (str): Course org filter, this value will be used to filter out the correct site configuration.
         name (str): Name of the key for which to return configuration value.
-        default: default value tp return if key is not found in the configuration
+        default: default value to return if key is not present in the configuration
 
     Returns:
         Configuration value for the given key.
 
     """
-    if is_site_configuration_enabled():
+    # Here we first look for the asked org inside site configuration, and if org is not present in site configuration
+    # then we go ahead and look it inside microsite configuration.
+    if SiteConfiguration.has_org(org):
```

Comment:
`SiteConfiguration.get_value_for_org` would return the `default` value for the following cases
1. Given `org` is not present in any of the `SiteConfiguration`
2. `SiteConfiguration` belonging to the given `org` does not contain an override for the provided `key`
and we want to fallback to `MicrositeConfiguration` only for the first case.